hadoop-ozone/dist/src/main/compose/ozone/test.sh

```diff
@@ -47,6 +47,8 @@ for bucket in link generated; do
   execute_robot_test scm -v BUCKET:${bucket} -N s3-${bucket} s3
 done
 
+execute_robot_test scm -v BUCKET:link -N s3-link botoclient
```

Comment:
I suggest:

 * moving `boto3.robot` and `boto_client.py` to `smoketest/s3`
 * removing this line
 * updating path of `boto_client.py` in `boto3.robot`
 * using `${BUCKET}` in `boto_client.py` for testing file upload, list bucket, etc.(most test cases except bucket creation) instead of `bucket1` created by the test

With that `boto3` test would get executed for all the environments and parameters where we run the complete `s3` suite of existing `awscli`-based tests (secure/unsecure, HA/non-HA, regular buckets and links).